src/auth/components/game-img/game-img.vue

```diff
@@ -0,0 +1,108 @@
+import { ThemeStore } from '../../../_common/theme/theme.store';
+<template>
+	<div id="cover-img-credits">
+		<div class="flow-col">
+			<a class="link-unstyled -title" :href="gameUrl">{{ gameTitle }}</a>
+			<a class="link-muted -dev" :href="devUrl"
+				>by <strong>{{ developerName }}</strong></a
+			>
+			<span class="text-muted -follows">{{ gameFollows }} followers</span>
+		</div>
+		<a class="-img" :href="gameUrl">
+			<app-game-thumbnail-img :game="coverGame" />
+		</a>
+	</div>
+</template>
+
+<style scoped>
+#cover-img-credits {
+	position: fixed;
+	right: 17px;
+	bottom: 0;
+	padding: 5px;
+	margin: 20px;
+
+	display: flex;
+
+	background: #262b2b;
+	border-radius: 8px;
+}
+
+#cover-img-credits a.-title:hover {
```

Comment:
since it's stylus, you would put these rules indented under the top-level class, however, since you named it .-title, just use that as the selector: `.-title:hover`